src/main/java/kostka/moviecatalog/service/CacheService.java

```diff
@@ -37,4 +39,21 @@ public CacheService(final RedisService redisService) {
         }
         return Collections.emptyList();
     }
+
+    public List<MovieListDto> getMovieDtosFromMoviesWithKey(final String key) {
```

Comment:
Here it is kinda confusing for me. Now you have MovieListDto, MovieFormDto, this method is called getMovieDto (did you mean MovieListDto? Personally I would not include return type in the method name)

Reply:
I have tried to rename the methods a little bit, can you check again please?